### PR TITLE
Erinhales/sdk bridge rc.13

### PR DESCRIPTION
**Pull request by ErinHales**
Hard code gas limit for sendNative

## Motivation

Solve out of gas error when sending native tokens

## PR Checklist

- [ ] Added Tests
- [ ] Updated Documentation
- [x] Updated CHANGELOG.md for the appropriate package

### CODE REVIEW

File: packages/sdk-bridge/CHANGELOG.md [hunk 1]
```diff
@@ -2,6 +2,8 @@
 
 ### Unreleased
 
+### 1.0.0-rc.13
+
```
Comment (Imti) [inline packages/sdk-bridge/CHANGELOG.md:6]: Prob add something about bumping the configuration package?

**Review by Imti — APPROVED** [review]
minor comment

File: packages/sdk-bridge/CHANGELOG.md (hunk 1 above)
Comment (yourbuddyconner) [reply packages/sdk-bridge/CHANGELOG.md:6]: We're not bumping the configuration package here, considering we will not be releasing `rc.19` to production just yet.

**Review by yourbuddyconner — APPROVED** [review]

File: packages/sdk-bridge/CHANGELOG.md (hunk 1 above)
Comment (ErinHales) [reply packages/sdk-bridge/CHANGELOG.md:6]: bumped to rc-18

**Pull request merged**